### PR TITLE
PartyMap type proposition

### PR DESCRIPTION
**Pull request by mathieud**

### CODE REVIEW

File: src/components/templates/PartyMap/PartyMapPage.tsx [hunk 1]
```diff
@@ -104,34 +116,33 @@ const PartyMap = () => {
...
             setSelectedRoom={setSelectedRoom}
             setIsRoomModalOpen={setIsRoomModalOpen}
           />
         </div>
         <div className="row">
           <div className="col">
-            <RoomList
-              startUtcSeconds={venue.start_utc_seconds}
-              rooms={venue.rooms}
+            {/* <RoomList
```
Comment (thecadams) [inline src/components/templates/PartyMap/PartyMapPage.tsx:132]: Should this be commented? Is this PR WIP?

File: src/components/templates/PartyMap/PartyMapPage.tsx [hunk 2]
```diff
@@ -104,34 +116,33 @@ const PartyMap = () => {
...
               attendances={attendances}
               setSelectedRoom={setSelectedRoom}
               setIsRoomModalOpen={setIsRoomModalOpen}
-            />
-          </div>
-          <div className="col-5 chat-wrapper">
-            <Chatbox />
+            /> */}
           </div>
+          <div className="col-5 chat-wrapper">{/* <Chatbox /> */}</div>
```
Comment (thecadams) [inline src/components/templates/PartyMap/PartyMapPage.tsx:140]: Again re commenting

File: src/hooks/useConnectCurrentVenue.ts [hunk 3]
```diff
@@ -14,6 +14,23 @@ const useConnectCurrentVenue = () => {
...
+      subcollections: [{ collection: "subvenues" }],
+      storeAs: "currentSubVenues",
+    },
+  ]);
+
+  useFirestoreConnect([
+    {
+      collection: "venues",
+      where: [["parentId", "==", venueId]],
+      storeAs: "currentChildVenues",
```
Comment (thecadams) [inline src/hooks/useConnectCurrentVenue.ts:31]: Too much complexity, let's scrap this parent/child thing, and only have subvenues.

In SparkleTown, different venues are "down the street from one another".

File: src/types/PartyMapVenue.ts [hunk 4]
```diff
@@ -16,15 +17,55 @@ export interface PartyMapVenue extends Venue {
...
+  descriptions?: string[];
+  duration_minutes: number;
+  // Should this be a subcollection?
+  // sub_venues: {
+  //   id: string;
+  //   schedule: PartyMapScheduleItem[];
+  // };
+}
+
+// this is roughlt EventData
```
Comment (thecadams) [inline src/types/PartyMapVenue.ts:63]: nit: tpyo

**Comment by thecadams** [comment]
My recommendation here is to remove the child venue thing, it's extra complexity we can live without.

We also need to get everything uncommented again before we can merge, of course

File: src/hooks/useConnectCurrentVenue.ts (hunk 3 above)
Comment (mathieud) [reply src/hooks/useConnectCurrentVenue.ts:31]: Ah, do you want to get rid of the `subvenue` collection within a venue and store all the information in the linked `Venue` document?

Comment (thecadams) [reply src/hooks/useConnectCurrentVenue.ts:31]: The opposite; ie. if there are sub-spaces within a venue they live in that venue.

**Comment by thecadams** [comment]
Closing outdated PRs <3

**Pull request closed without merging**